_config.yml

```diff
@@ -16,7 +16,7 @@
 title: Holochain
 email: help@holo.host
 description: >- # this means to ignore newlines until "baseurl:"
-  Holochain site description
+  Holochain site descriptionHolographic storage for distributed applications -- a validating monotonic DHT "backed" by authoritative hashchains for data provenance
```

Comment:
It looks like you left in the old site description!

Reply:
Updated and added logo and links in new commit